mon-pix/app/templates/components/form-textfield.hbs

```diff
@@ -38,5 +38,5 @@
     </div>
 </div>
 
-<div id="validationMessage-{{textfieldName}}" class="form-textfield__message {{validationMessageClass}} form-textfield__message-{{textfieldType}}" role="alert">{{#if displayMessage}}{{validationMessage}}{{/if}}</div>
+<div id="validationMessage-{{textfieldName}}" class="form-textfield__message {{validationMessageClass}} form-textfield__message-{{textfieldType}}" aria-live="polite">{{#if displayMessage}}{{validationMessage}}{{/if}}</div>
```

Comment:
**suggestion**
Idem ici, je pense qu'on peut remplacer la balise `div`, en mettant un `label` et `for` si ça a du sens à le rattacher à un champ ? Ou alors un `p` peut être ?